### DISCUSSION TITLE
REPORTE AQUI BUG DO PROJETO DA API DE CADASTRO GERAL - TECH CHALLENGE

### DISCUSSION THREAD

**Discussion by MichelKelvin** [community/community#60162] [Pull Requests]
### Select Topic Area

Bug

### Body

Descreva seus problemas ao utilizar a nossa API e contribua nos ajudando!

**Comment by Michel-Kelvin** [comment]
Olá, não tenho nenhum bug a relatar, sua API é muito boa! Parabéns!

**Comment by gustavoLimaOliveira** [comment]
Parabéns pelo projeto pessoal , nota máxima
